docs/odata/v2vsv4.markdown

```diff
@@ -0,0 +1,48 @@
+
+* $count replaces `$inlinecount` in OData 4. `$count` has been enhanced to be used with `$filter`, `$expand` and `$orderby` options.
+
+* Changes in support for data types in OData 4:
+  * `Edm.DateTime` has been deprecated. The lack of timezone information in OData 2 causes significant problems. Use `Edm.DateTimeOffset` instead.
+  * `Edm.Time` has been replaced with `Edm.Duration` and `Edm.TimeOfDay` to make it clear whether it is duration of a specific time of day.
+  * `Edm.Date` has been added as there was no way to express just a date in OData 2.
+  * `Edm.Float` has been eliminated.
+
+* The oData V4 model does not support the methods `getData`, `getObject`, `getOriginalProperty`, `getProperty`. For data access, use the context API instead of methods on the model.
+
+* The oData V4 model does not support the methods `getChangeBatchGroups`, `getChangeGroups`, `getDeferredGroups`, `setChangeBatchGroups`, `setChangeGroups`, `setDeferredBatchGroups`, `setDeferredGroups`, `setUseBatch` (and corresponding model construction parameters). Batch groups are solely defined via binding parameters with the corresponding parameters on the model as default. Application groups are by default deferred; there is no need to set or get deferred groups. You just need the `submitBatch` method on the model to control execution of the batch. You can use the predefined batch group `$direct` to switch off batch either for the complete model or for a specific binding (only possible for the complete model in V2).
+
+* OData operations executed via binding: The oData V4 model does not support the method `callFunction`. Use an operation binding instead.
+
+* No CRUD methods on model: The oData V4 model does not support the methods `create`, `read`, `remove`, `update`. `read`, `update`, `create` and `remove` operations are available implicitly via the bindings, so that changes are bound to controls. It is not possible to trigger requests for specific OData URLs.
+
+* No metadata access via model: The oData V4 model does not support methods `getServiceAnnotations`, `getServiceMetadata`, `refreshMetadata` as well as methods corresponding to the events `metadataFailed`, `metadataLoaded`. Metadata is only accessed via `ODataMetaModel`. Metadata is only loaded when needed (e.g. for type detection or to compute URLs for write requests); the corresponding methods on the `v4.ODataMetaModel` use promises instead of events.
+
+* `sap.ui.model.odata.AnnotationHelper` is not supported for OData V4.
+
+---
+
+## NOTE
```

Comment:
many good points in here!
probably putting them in separate paragraphs might be easier on the eyes and help grasp the content quicker.